app/src/Demo/Domain/Service/Administrator/SetActive.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Demo\Domain\Service\Administrator;
+
+use Demo\Domain\Model\Administrator\AdministratorDto;
+use Demo\Domain\Model\Administrator\AdministratorRepository;
+use Demo\Domain\Service\Administrator\SetActiveInterface;
+use Ivoz\Core\Domain\Service\EntityTools;
+
+class SetActive implements SetActiveInterface
+{
+    public function __construct(
+        private AdministratorRepository $administratorRepository,
+        private EntityTools $entityTools
+    ) {
+    }
+
+    public function execute(int $administratorId): void
+    {
+        $administrator = $this->administratorRepository->find($administratorId);
+
+        // I don’t think we should launch an exception in this case.
```

Comment:
How do we know if the action has gone well?

Reply:
You are rigth. I was overthinking it in terms of security, which I should have avoided.